prometheus/cluster-role-binding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
```

Comment:
I got this error on `tilt up`:
```
Build Failed: kubectl apply:
stderr: The ClusterRoleBinding "prometheus" is invalid: subjects[0].namespace: Required value
: exit status 1
```

which went away when I added a `namespace: default` here

Reply:
oops, you're right, i messed that up when i was cleaning up the pr for general use